guide/inheritance-wallet/onboarding-cosigners.md

```diff
@@ -0,0 +1,156 @@
+      alt: Placholder screen representing the device-specific wallet activation flow.
+      caption: The app guides her throught the import flow on her BitBox.
+    - file: alice-enable-device/enable-alice-success
+      alt: Success screen showing the option activate another device or go to the home screen.
+      caption: On the success screen, the app shows the option to enable another signing device.
+    - file: alice-enable-device/enable-alice-completion
+      alt: Home screen showing the wallet with two tasks associated.
+      caption: On the home screen, Alice checks off the task. 
+    - file: alice-enable-device/enable-alice-completion-dialog
+      alt: Screen showing a confirmation dialog.
+      caption: The app asks whether she has enabled all signing devices.
+    - file: alice-enable-device/enable-alice-done
+      alt: Home screen showing the wallet with only one task associated.
+      caption: The last thing to do is to download the wallet backup kit.
+---
+
+<!--
+
+Editor's notes
+
+This page covers how Alice imports the savings wallet to her own app and enables her signing device to create and sign transactions.  
+
+Illustration sources
+
+https://www.figma.com/file/h5GP5v5dYfpXXfEUXf6nvC/Family-inheritance-wallet?type=design&node-id=5542%3A2119&mode=design&t=sBtcvrDzb8MPtWaK-1
+
+-->
+
+# Cosigner onboarding
+{: .no_toc }
+
+---
+
+<div class="glossary-toc" markdown="1">
+ * Table of contents
+{:toc}
+</div>
+
+---
+
+[So far]({{ '/guide/inheritance-wallet/wallet-creation/' | relative_url }}), the wallet has been created on Bob's app. The next step is to get Alice set up as a cosigner and to enable her to sign transactions. To do this Alice needs to:
+
+1. Import the wallet configuration to her app.
+2. Enable her BitBox to sign transactions. 
+
+### Wallet onboarding
+The first step is to add the family savings wallet on Alice's app.
+
+#### Bob shares the wallet configuration
+Bob shares the wallet configuration with Alice by tapping on the "onboard cosigner" task. Just like during wallet creation, this displays a QR code that Alice can scan to import the configuration. Alternatively, Bob can share the configuration over a secure channel.
+
+{% include image-gallery.html pages = page.images_bob-export-config %}
+
+The app also shows an option to view the [wallet descriptor](https://bitcoin.design/guide/glossary/#output-script-descriptor). The wallet descriptor is a small piece of text that contains all the information that a wallet application needs to recover a multi-key wallet. We will cover this topic in more detail in the chapter about the wallet backup.
+
+After Alice has successfully set up the wallet on her end, Bob checks off the task. For our application, this is a manual step, because it does not run a server that coordinates activity between the users. This adds a bit more friction to the user experience but minimizes reliance on the wallet provider.
+
+#### Alice sets up the wallet on her app
```

Comment:
```suggestion
#### Alice joins as cosigner
```

Reply:
Not sure this is the best suggestion, but there is an opportunity to simplify and be more consistent when talking about this step. It is variously described as:
- onboarding
- wallet setup / sets up wallet
- add / import wallet
- join as cosigner

As far as I understand it is one step, not several. So finding the best description might make the whole page easier to read. If it was to be called 'Alice joins as cosigner' or similar, then I might separate the 'Recover wallet' and 'Join as cosigner' in the Add wallet modal image/flow.

This is just a suggestion, you decide.

Reply:
Yeah, wording is tricky. I went with "import wallet", as this is what really happens. Users don't really "join" a wallet. Plus, if I use the term join, then I still need to explain that in order to join, the wallet needs to be imported 🙂